autopush/router/simple.py

```diff
@@ -81,13 +83,17 @@ def route_notification(self, notification, uaid_data):
             try:
                 result = yield self._send_notification(uaid, node_id,
                                                        notification)
+            except CancelledError as exc:
+                # Log the timeout, but don't clear the node.
+                self.log.debug("Could not route message: Timedout {exc}",
+                               exc=exc)
             except (ConnectError, ConnectionClosed, ResponseFailed) as exc:
                 self.metrics.increment("updates.client.host_gone")
                 yield deferToThread(router.clear_node,
                                     uaid_data).addErrback(self._eat_db_err)
-                if isinstance(exc, ConnectionRefusedError):
+                if isinstance(exc, (ConnectionRefusedError, CancelledError)):
```

Comment:
CancelledError never occurs here with the new except block above

I think we now want some additional CancelledError handling around _send_notification_check as well, though